### DISCUSSION TITLE
How do I set a profile picture/avatar?

### DISCUSSION THREAD

**Discussion by SinTan1729** [requarks/wiki#6534] [Help / Questions] [answered]
I'm new to Wiki.js and this might be a very stupid question, but how do I set a profile picture/avatar for my user? I'm unable to find any option to do that.

**Comment by NGPixel** [comment, marked as answer]
This is only possible when using an authentication strategy that provides an avatar URL (e.g. google login).

The upcoming v3 will have the option to upload an avatar, regardless of the authentication strategy.

**Reply by SinTan1729** [reply]
That's good to know. I want to keep it local, so google login isn't an option. Waiting for v3.
